webapp/channels/src/components/post_view/scroll_to_bottom_arrows.tsx

```diff
@@ -5,31 +5,31 @@ import React from 'react';
 
 import ScrollToBottomIcon from 'components/widgets/icons/scroll_to_bottom_icon';
 
-interface Props {
+type Props = {
     isScrolling: boolean;
     atBottom?: boolean;
     onClick: () => void;
 }
 
-export default class ScrollToBottomArrows extends React.PureComponent<Props> {
-    render() {
-        // only show on mobile
-        if (window.innerWidth > 768) {
-            return null;
-        }
-
-        let className = 'post-list__arrows';
-        if (this.props.isScrolling && this.props.atBottom === false) {
-            className += ' scrolling';
-        }
+const ScrollToBottomArrows = ({ isScrolling, atBottom, onClick }:Props) => {
+    // only show on mobile
+    if (window.innerWidth > 768) {
+        return null;
+    }
 
-        return (
-            <div
-                className={className}
-                onClick={this.props.onClick}
-            >
-                <ScrollToBottomIcon/>
-            </div>
-        );
+    let className = 'post-list__arrows';
+    if (isScrolling && atBottom === false) {
+        className += ' scrolling';
     }
```

Comment:
Sorry for the back and forth with this one. In order to simplify this, you can use the `classNames` function. This would end up something like this:
```suggestion
    const className = classNames('post-list__arrows', {scrolling: isScrolling && (atBottom === false)});
```

If we do it like this, we could even consider having it inline in the component in line 27. I.e.
```
<div
    className={classNames(...)}
    ...
```